apps/ledger-live-mobile/src/const/navigation.js

```diff
@@ -231,9 +236,34 @@ export const ScreenName = {
   CosmosClaimRewardsValidationError: "CosmosClaimRewardsValidationError",
   CosmosClaimRewardsValidationSuccess: "CosmosClaimRewardsValidationSuccess",
 
+  OsmosisDelegationStarted: "OsmosisDelegationStarted",
```

Comment:
Are these Osmosis-related additions intended?

Reply:
these were to resolve the merge conflicts reported by Github below, however it doesn't look like Git refreshed the checks.